src/styles/blocks/card.scss

```diff
@@ -0,0 +1,108 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 0 16px 16px;
+
+  border: 1px solid  #f3f3f3;
+
+  &__logo {
+    margin: 32px auto 40px;
+    @include logoSize();
+  }
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &:nth-last-child(-n + #{$card-count - 4}) {
+    margin-top: 46px;
+  }
+
+  &__title {
+    margin-top: 0;
+    margin-bottom: 4px;
+
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: #060b35;
+  }
+
+  &__code {
+    margin: 0 0 16px;
+
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+
+    color: #616070;
+  }
+
+  &__rating {
+    margin-bottom: 24px;
+
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    &-reviews {
+      margin: 0;
+      font-size: 10px;
+      font-weight: 400;
+      color: #060b35;
+    }
+  }
+
+  &__price {
+    margin-bottom: 16px;
+
+    display: flex;
+    justify-content: space-between;
+
+    line-height: 18px;
+
+    &-text {
+      margin: 0;
+
+      font-size: 12px;
+      font-weight: 400;
+
+      color: #616070;
+    }
+
+    &-amount {
+      margin: 0;
+
+      font-size: 16px;
+      font-weight: 700;
+      color: #060b35;
+    }
+  }
+
+  &__button {
+    display: flex;
+    text-decoration: none;
+    justify-content: center;
+    align-items: center;
+    height: 40px;
+
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    color: #fff;
+
+    background-color: #00acdc;
+    border: 1px transparent;
+    border-radius: 5px;
+
```

Comment:
same here. try to use code style. https://codeguide.academy/html-css.html#css-order